src/MgvLib.sol

```diff
@@ -96,14 +96,14 @@ interface HasMgvEvents {
   /* 
     Events in solidity is a hard thing to do in a optimal way. If you look at it as a purely gas efficient issue, you want to emit as few events as possible and with as few fields as possible. But as events also has to be usable for an off chain user, then doing this is not always the best solution.
 
-    We tried to list the main points that we would like to do with events.
+    We tried to list the main forces that are inflicted upon events.
```

Comment:
"inflict" sounds somewhat ominous 😅

Maybe this is better?

```suggestion
    We tried to list the main forces that drive which events and data to emit:
```

Reply:
I added your suggestions to feat/doc pr since you got front-run

Reply:
Thanks and darn